### ISSUE TITLE
Quick question about scrubbing/synonyms

### ISSUE THREAD

**Issue by lardconcepts** [terrenjpeterson/caloriecounter#77]
Hi there; I'm new to Lex, having from from the Alexa world, and really impressed with this repo.
Learning a lot from your code, but I have a question:

At https://github.com/terrenjpeterson/caloriecounter/blob/master/src/lambda.js#L198, you "scrub the restaurant name to get it to a standard".

Is there any particular advantage of doing it this way, over using [synonyms](https://aws.amazon.com/blogs/machine-learning/use-synonyms-and-slot-value-validation-in-your-amazon-lex-chatbots/)?

> With the synonym functionality, you can specify multiple synonyms for a slot value in your chatbot. The synonyms specified are resolved to the corresponding slot values. For example, if the slot value is “comedy”, with “funny” and “humorous” specified as synonyms, then user input of “funny” will now be resolved to “comedy”. You can optimize your business logic to implement code for just the slot value instead of entire set of synonyms. In addition, the chatbot automatically resolves minor variants of the slot value to the original value without the user having to explicitly specify the synonym. For example, if “pineapple” is one of the slot values and the user value is “pineapples” then the slot is resolved to the value “pineapple”.

Over in the Alexa world, when anything is updated (even synonyms) the whole skill has to go through a laborious and often random re-approval process, so people often do lambda-side stuff like you've done, to keep it away from the front-end checking.

But (as far as I understand it) that's no needed here? It's quite possible (almost certain!) that I've completely mis-understood something here, so if anyone can set me straight I'd be grateful. Thanks!

**Comment by terrenjpeterson** [comment]
Good idea.  It would be great to simplify the code and have this be embedded into the model vs. the functions.
